src/amo/containers/App.js

```diff
@@ -9,17 +9,18 @@ import { compose } from 'redux';
 import 'core/fonts/fira.scss';
 import 'amo/css/App.scss';
 import SearchForm from 'amo/components/SearchForm';
+import ErrorPage from 'amo/components/ErrorPage';
```

Comment:
I should have mentioned that the reason I like it is you can read the `<ErrorPage>` code (i.e. where the component is used) more naturally.